packages/msw/mixin.browser.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-env browser */
+
+import { Mixin } from 'hops-mixin';
+
+const registerBrowserMock = (
+  { graphql, rest },
+  { type, method, identifier, data }
+) => {
+  if (type === 'graphql') {
+    return graphql[method](identifier, (_, res, ctx) => res(ctx.data(data)));
+  }
+
+  if (type === 'rest') {
+    return rest[method](identifier, (_, res, ctx) =>
+      res(
+        ctx.status(200),
+        typeof data === 'undefined' ? undefined : ctx.json(data)
+      )
+    );
+  }
```

Comment:
The `graphql[method]` and `rest[method]` accesses are looking a little bit dangerous. I think it could be easy to write a wrong mock that would cause a runtime error here. I think having a helpful error message instead would be great in such a case.

The two `if`s also look weird / could be put in a `if/else-if/else`-clause to also detect invalid mock types.

```js
  if (type === 'graphql') {
    // ...
  } else if (type === 'rest') {
    // ...
  } else {
    // warn about unhandled mock request
  }
```